docs/htdocs/info/docs/tools/vep/script/vep_plugins.html

```diff
@@ -80,46 +80,48 @@ <h2 id="plugins_existing">Existing plugins</h2>
ortant;height:18px" onclick="show_hide('disgenet');" id="a_disgenet">more</a></p><div id="div_disgenet" style="display:none;"><p>Please cite the DisGeNET publication alongside the VEP if you use this resource:<br />https://academic.oup.com/nar/article/48/D1/D845/5611674</p><p>Options are passed to the plugin as key=value pairs:</p><p>file           : Path to DisGENET data file (mandatory).</p><p>disease        : Set value to 1 to include the diseases/phenotype names reporting<br />the Variant-PMID association (optional).</p><p>rsid           : Set value to 1 to include the dbSNP variant Identifier (optional).</p><p>filter_score   : Only reports citations with score greater or equal than input value (optional).</p><p>filter_source  : Only reports citations from input sources (optional).<br />Accepted sources are: UNIPROT, CLINVAR, GWASDB, GWASCAT, BEFREE<br />Separate multiple values with '&'.</p><p>unique         : Only reports unique dbSNP variant Identifiers and diseases/phenotype names (optional)</p><p>Output:<br />The output includes: <br />- PMID of the publication reporting the Variant-Disease association (default)<br />- DisGENET score for the Variant-Disease association (default)<br />- dbSNP variant Identifier (optional)<br />- diseases/phenotype names (optional)</p><p>The following steps are necessary before running this plugin:<br />This plugin uses file 'all_variant_disease_pmid_associations.tsv.gz'<br />File can be downloaded from: https://www.disgenet.org/downloads</p><p>gunzip all_variant_disease_pmid_associations.tsv.gz</p><p>awk '($1 ~ /^snpId/ || $2 ~ /NA/) {next} {print $0 | "sort -k2,2 -k3,3n"}' <br />all_variant_disease_pmid_associations.tsv > all_variant_disease_pmid_associations_sorted.tsv</p><p>bgzip all_variant_disease_pmid_associations_sorted.tsv<br />tabix -s 2 -b 3 -e 3 all_variant_disease_pmid_associations_sorted.tsv.gz</p><p>The plugin can then be run as default:<br /><pre class="code sh_sh"> ./vep -i variations.vcf --plugin DisGeNET,file=all_variant_disease_pmid_associations_sorted.tsv.gz<br /></pre><br />or with an option to include optional data or/and filters: <br /><pre class="code sh_sh"> ./vep -i variations.vcf --plugin DisGeNET,file=all_variant_disease_pmid_associations_sorted.tsv.gz,<br /></pre>disease=1<br /><pre class="code sh_sh"> ./vep -i variations.vcf --plugin DisGeNET,file=all_variant_disease_pmid_associations_sorted.tsv.gz,<br /></pre>disease=1,filter_source='GWASDB&GWASCAT'</p><p>Of notice: this plugin only matches the chromosome and the position in the<br />chromosome, the alleles are not taken into account to append the DisGENET data.<br />The rsid is provided (optional) in the output in order to help to filter the relevant data.</p></div></td><td><div class="vdoc_dtype_count" style="float:left;padding:2px 6px;cursor:default;background-color:#22949B">Phenotype data</div></td><td><a href="https://metacpan.org/pod/List::MoreUtils" rel="external">List::MoreUtils</a> qw(uniq)</td><td>Ensembl</td></tr>
```

Comment:
Category should be "phenotype_data" rather than "phenotype-data" for DisGeNET. 
Do check the category menu is returning DisGeNET as a phenotype resource and 'ND' is not present.